airsonic-main/src/main/java/org/airsonic/player/service/SettingsService.java

```diff
@@ -86,6 +86,7 @@
     private static final String KEY_WELCOME_SUBTITLE = "WelcomeSubtitle";
     private static final String KEY_WELCOME_MESSAGE = "WelcomeMessage2";
     private static final String KEY_LOGIN_MESSAGE = "LoginMessage";
+    private static final String KEY_SESSION_TIMEOUT_MESSAGE = "SessionTimeout";
```

Comment:
Probably should be named `server.servlet.session.timeout` so it overrides the spring boot value